### ISSUE TITLE
Fix grammar warnings from black (non-fatal, just annoying)

### ISSUE THREAD

**Issue by machow** [cal-itp/reports#43]
I'm not sure why our report building prints out a warning from black (TODO: @tiffanychu90 has it copied). It doesn't affect our report, but is annoying.

**Comment by tiffanychu90** [comment]
When executing 1 report:
```
jovyan@jupyter-tiffanychu90:~/reports/reports$ make outputs/2021/09/206/index.htmlpapermill -f outputs/2021/09/206/parameters.json report.ipynb outputs/2021/09/206/index.ipynb
/opt/conda/lib/python3.9/site-packages/papermill/iorw.py:50: FutureWarning: pyarrow.HadoopFileSystem is deprecated as of 2.0.0, please use pyarrow.fs.HadoopFileSystem instead.
  from pyarrow import HadoopFileSystem
Input Notebook:  report.ipynb
Output Notebook: outputs/2021/09/206/index.ipynb
Generating grammar tables from /opt/conda/lib/python3.9/site-packages/blib2to3/Grammar.txt
Writing grammar tables to /home/jovyan/.cache/black/21.8b0/Grammar3.9.6.final.0.pickle
Writing failed: [Errno 2] No such file or directory: '/home/jovyan/.cache/black/21.8b0/tmpz88bumlc'
Generating grammar tables from /opt/conda/lib/python3.9/site-packages/blib2to3/PatternGrammar.txt
Writing grammar tables to /home/jovyan/.cache/black/21.8b0/PatternGrammar3.9.6.final.0.pickle
Writing failed: [Errno 2] No such file or directory: '/home/jovyan/.cache/black/21.8b0/tmp8vpyfj4k'
Executing:   0%|                                         | 0/31 [00:00<?, ?cell/s]Executing notebook with kernel: python3
Executing: 100%|████████████████████████████████| 31/31 [00:35<00:00,  1.16s/cell]
jupyter nbconvert \
        --to html \
        --TagRemovePreprocessor.remove_input_tags="{'hide', 'injected-parameters'}" \
        --TagRemovePreprocessor.remove_single_output_tags="{'hide', 'injected-parameters'}" \
        --no-input \
        --no-prompt \
        outputs/2021/09/206/index.ipynb
/opt/conda/lib/python3.9/site-packages/traitlets/traitlets.py:2556: FutureWarning: --TagRemovePreprocessor.remove_input_tags={'hide', 'injected-parameters'} for containers is deprecated in traitlets 5.0. You can pass `--TagRemovePreprocessor.remove_input_tags item` ... multiple times to add items to a list.
  warn(
/opt/conda/lib/python3.9/site-packages/traitlets/traitlets.py:2556: FutureWarning: --TagRemovePreprocessor.remove_single_output_tags={'hide', 'injected-parameters'} for containers is deprecated in traitlets 5.0. You can pass `--TagRemovePreprocessor.remove_single_output_tags item` ... multiple times to add items to a list.
  warn(
[NbConvertApp] Converting notebook outputs/2021/09/206/index.ipynb to html
[NbConvertApp] Writing 648832 bytes to outputs/2021/09/206/index.html 
```

**Comment by acouch** [comment]
This is an issue with black identified here: https://github.com/psf/black/issues/2681

I've tried the fix here: https://github.com/psf/black/issues/2681#issuecomment-1069541949 which works, however it is hard to have a universal fix since the make commands could be run from local or in docker. If we want to fix for just docker we could try adding that to the docker file.

There is a way to remove black now with papermill:
https://github.com/nteract/papermill/pull/631/files#diff-212b1698766800b9f9593278bb7275d8e5b9d859ef85f6acbc4927ed3d8660b9R195 however that was before the last release: https://github.com/nteract/papermill/releases/tag/2.3.3

**Comment by acouch** [comment]
Actually there is a new release: https://github.com/nteract/papermill/issues/642#issuecomment-1019371667 It is just not on github. Will test updating & removing black to see if it solves this issue.

**Comment by holly-g** [comment]
Resolved by #152
